src/main/java/com/tlswe/awsmock/ec2/model/DefaultMockEc2Instance.java

```diff
@@ -15,7 +15,7 @@
      *
      * @see Serializable
      */
-    private static final long serialVersionUID = 1L;
+    private static final long serialVersionUID = 2L;
```

Comment:
Could you explain why the version is incremented for this class?

Reply:
You are right, the base model would be a better place for the increment.

Reply:
Yeah not necessary a change here.